editor/project_manager.cpp

```diff
@@ -44,6 +44,7 @@
 #include "editor_scale.h"
 #include "editor_settings.h"
 #include "editor_themes.h"
+#include "main/main.h"
```

Comment:
This should be moved out of `main`, we shouldn't include `main.h` unless strictly necessary.